fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/DefaultPhysicalTableResolver.java

```diff
@@ -51,30 +49,32 @@ public DefaultPhysicalTableResolver(
     }
 
     @Override
-    public List<PhysicalTableMatcher> getMatchers(DataApiRequest apiRequest, TemplateDruidQuery query) {
-        SchemaPhysicalTableMatcher schemaMatcher = new SchemaPhysicalTableMatcher(
-                apiRequest,
-                query,
-                resolveAcceptingGrain.apply(apiRequest, query)
-        );
-        TimeAlignmentPhysicalTableMatcher timeAlignmentMatcher = new TimeAlignmentPhysicalTableMatcher(apiRequest);
-        AggregatableDimensionsMatcher aggregatabilityMatcher = new AggregatableDimensionsMatcher(apiRequest, query);
+    public List<PhysicalTableMatcher> getMatchers(QueryPlanningConstraint requestConstraints) {
+        SchemaPhysicalTableMatcher schemaMatcher =
+                new SchemaPhysicalTableMatcher(requestConstraints);
```

Comment:
Trying to keep a nice pattern here so that the pattern can be easily recognized. Let me know otherwise.